src/openforms/payments/contrib/ogone/plugin.py

```diff
@@ -99,10 +103,11 @@ def handle_webhook(self, request):
         except InvalidSignature as e:
             logger.warning(f"invalid SHASIGN for payment {payment}")
             logevent.payment_flow_failure(payment, self, e)
-            return HttpResponseBadRequest("bad shasign")
+            return HttpResponseBadRequest("bad shasign"), None
```

Comment:
I see a mix of raise 404 (`get_object_or_404`) and return `HttpResponseBadRequest`.

Shouldn't we choose just to raise an exception instead of return `HttpResponseBadRequest`? This way, we also don't get the ackward tuple, and simply catch the exception in the view to return a `HttpResponseBadRequest`?